hindi/actor4/aamir.md

```diff
@@ -0,0 +1,2 @@
+आप अमीर खान की गोद में खेल रहे हैं|  मज़ा आ रहा है ?
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.